webextension/privileged/ui/CloudDownloadsView.jsm

```diff
@@ -533,6 +565,13 @@ var CloudDownloadsView = {
         CloudDownloadsInternal.selectedProvider = { providerKey, value: this.providers.get(providerKey) };
         CloudDownloadsInternal.handleMove(download);
       }
+      telemetryData = {
+        message: "prompt_move_download_context_menu",
+        provider: providerKey,
+        provider_count: this.providers.size.toString(),
+        timestamp: Math.floor(Date.now() / 1000).toString(),
```

Comment:
Why do you need a timestamp that's this granular?

Reply:
Started storing last prompt time inside cloud.services.lastPrompt pref in seconds and to keep the format consistent, kept  time in seconds across Cloud Storage

Reply:
OK, but then, why do you need a timestamp in the telemetry at all? And why couldn't it just be, say, the date?

Getting second-granularity timestamps feels like something that has privacy implications. Please think carefully about whether we actually need that for the study we're doing.

Reply:
sounds good, updated time stamp to use new Date()